app/decorators/proposal_decorator.rb

```diff
@@ -9,26 +9,8 @@ def total_approvers
     object.individual_steps.count
   end
 
-  def steps_by_status
-    # Override default scope
-    object.individual_steps.with_users.reorder(
-      # http://stackoverflow.com/a/6332081/358804
-      <<-SQL
-        CASE steps.status
-        WHEN 'approved' THEN 1
-        WHEN 'actionable' THEN 2
-        ELSE 3
-        END
-      SQL
-    )
-  end
-
   def steps_in_list_order
```

Comment:
`steps_in_list_order` is used in the emails. I just created a proposal on master with a parallel flow and 2 steps, this is what the email looks like:

![screen shot 2015-12-16 at 10 13 12 am](https://cloud.githubusercontent.com/assets/601515/11849641/ad1d79e4-a3dd-11e5-8c6f-1b4552be52dc.png)

seems like this method is not working